.npmrc

```diff
@@ -0,0 +1,2 @@
+frozen-lockfile=true
+node-linker=hoisted
```

Comment:
Instead of storing this configuration in a separate file, add the following options to `pnpm-workspace.yaml`:

```
shamefullyHoist: true
preferFrozenLockfile: true
```

You can use the following PR as a reference: https://github.com/ckeditor/ckeditor5/pull/18966/files

Reply:
I am using the options that we agreed on during the `pnpm` meeting.